packages/workflow/src/workflow.ts

```diff
@@ -1290,3 +1316,9 @@ export const log: LoggerSinks['defaultWorkerLogger'] = Object.fromEntries(
     ];
   })
 ) as any;
+
+function assertInWorkflowContext(message: string): void {
```

Comment:
nit: this assertion is always called before `getActivator`.
You can return the activator from this function.